packages/debugger-extension/src/index.ts

```diff
@@ -228,6 +240,17 @@ const files: JupyterFrontEndPlugin<void> = {
       translator: translator
     });
 
+    const trans = translator.load('jupyterlab');
+
+    handler.executionDone.connect(() => {
+      void debug.displayModules().catch(reason => {
+        void showErrorMessage(
+          trans.__('Fail to get kernel sources'),
+          trans.__('Fail to get kernel sources:\n%2', reason)
+        );
```

Comment:
I have mixed feelings about this. Before, it made sense to show an error message since the refresh action was triggered by the user. 

But now that it's an automatic thing on every cell execution, it may get pretty bad in terms of UX if the debugger is not behaving. 

Maybe we should just `console.error` in that case.

Reply:
Same comment for the other places doing it.

Reply:
Sure, done in [f72b27a](https://github.com/jupyterlab/jupyterlab/pull/18081/commits/f72b27ab1957208af0943d9e8e409779066d72eb)